core/voting_table_test.go

```diff
@@ -1,3 +1,117 @@
 package core
 
-// @TODO (rgeraldes) - complete
+import (
+	"testing"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/kowala-tech/kUSD/core/types"
+	"github.com/kowala-tech/kUSD/common"
+	"math/big"
+	"github.com/stretchr/testify/require"
+	"github.com/kowala-tech/kUSD/core/types/mocks"
+)
+
+func TestTwoThirdsPlusOneVoteQuorum(t *testing.T) {
+	testCases := []struct {
+		voters    int
+		votes     int
+		hasQuorum bool
+	}{
+		{3, 2, false},
+		{3, 2, false},
+		{3, 3, true},
+		{10, 6, false},
+		{10, 7, true},
+		{100, 1, false},
+		{100, 66, false},
+		{100, 67, true},
+		{100, 100, true},
+	}
+	for _, tc := range testCases {
+		t.Run(fmt.Sprintf("voters %d votes %d quorum %t", tc.voters, tc.votes, tc.hasQuorum), func(t *testing.T) {
```

Comment:
I like this approach to naming the test - I usually add a name/context field but this makes more sense in some situations.